respa_admin/views/resources.py

```diff
@@ -473,17 +483,22 @@ def forms_invalid(
     def _validate_forms(
         self, form, period_formset, image_formset, resource_accessibility_formset
     ):
-        valid_form = form.is_valid()
-        valid_period_form = period_formset.is_valid()
-        valid_image_formset = image_formset.is_valid()
-        valid_resource_accessibility_formset = resource_accessibility_formset.is_valid()
-
-        return (
-            valid_form
-            and valid_period_form
-            and valid_image_formset
-            and valid_resource_accessibility_formset
-        )
+        try:
+            valid_form = form.is_valid()
```

Comment:
If the formsets include non input errors e.g. management forms have incorrect number of forms, `ValidationError` is raised in the `is_valid()` method, so we can catch here to prevent a 500 error.